x-pack/plugins/cases/server/services/cases/index.ts

```diff
@@ -614,7 +619,57 @@ export class CasesService {
 
       return { ...res, attributes: decodedRes };
     } catch (error) {
-      this.log.error(`Error on POST a new case: ${error}`);
+      this.log.error(`Error on creating a new case: ${error}`);
+      throw error;
+    }
+  }
+
+  public async bulkCreateCases({
+    cases,
+    refresh,
+  }: BulkCreateCasesArgs): Promise<SavedObjectsBulkResponseWithErrors<CaseTransformedAttributes>> {
+    try {
+      this.log.debug(`Attempting to bulk create cases`);
+
+      const bulkCreateRequest = cases.map(({ id, ...attributes }) => {
+        const decodedAttributes = decodeOrThrow(CaseTransformedAttributesRt)(attributes);
+
+        const { attributes: transformedAttributes, referenceHandler } =
+          transformAttributesToESModel(decodedAttributes);
+
+        transformedAttributes.total_alerts = -1;
+        transformedAttributes.total_comments = -1;
+
+        return {
+          type: CASE_SAVED_OBJECT,
+          id,
+          attributes: transformedAttributes,
+          references: referenceHandler.build(),
+        };
+      });
+
+      const bulkCreateResponse =
```

Comment:
I'm missing a test to check the right behaviour when the user wants to create two cases but the second one fails

Reply:
There is a test in `x-pack/plugins/cases/server/services/cases/index.test.ts` called `return cases with the SO errors correctly` that tests this behavior. Do you have something else in mind?